src/components/TodoCard/TodoCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../../app/store';
+import { setSelectedTodo } from '../../features/currentTodo';
+
+type Props = {
+  todo: Todo;
+  showModalWindow: (show: boolean) => void;
+  isShowModal: boolean;
+};
+
+const TodoCard: React.FC<Props> = ({ todo, showModalWindow, isShowModal }) => {
+  const dispatch = useDispatch<AppDispatch>();
+  const selectedTodo = useSelector((state: RootState) => state.currentTodo);
+
+  const isSelectedTodo = selectedTodo?.id === todo.id;
+
+  const handleSelectTodo = (selectedElement: Todo) => {
+    dispatch(setSelectedTodo(selectedElement));
+  };
+
+  return (
+    <tr
+      data-cy="todo"
+      className={cn(
+        selectedTodo?.id === todo?.id && isShowModal
+          ? 'has-background-info-light'
+          : '',
+      )}
+    >
+      <td className="is-vcentered">{todo.id}</td>
+      <td className="is-vcentered">
+        {todo.completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
+      </td>
+      <td className="is-vcentered is-expanded">
+        <p
+          className={cn({
+            'has-text-danger': !todo.completed,
+            'has-text-success': todo.completed,
+          })}
+        >
+          {todo.title}
+        </p>
+      </td>
+      <td className="has-text-right is-vcentered">
```

Comment:
i think u need more spaces)Change it in all the cases
```suggestion
      <td className="is-vcentered">{todo.id}</td>
      
      <td className="is-vcentered">
        {todo.completed && (
          <span className="icon" data-cy="iconCompleted">
            <i className="fas fa-check" />
          </span>
        )}
      </td>
      
      <td className="is-vcentered is-expanded">
        <p
          className={cn({
            'has-text-danger': !todo.completed,
            'has-text-success': todo.completed,
          })}
        >
          {todo.title}
        </p>
      </td>
      
      <td className="has-text-right is-vcentered">
```

Reply:
Thank you, got it, but my linter changes this all the time when I'm trying to make some space in code 👨‍💻